### PR TITLE
Add ADHD page

### PR DESCRIPTION
**Pull request by alvarlagerlof**
This should not be merged until the page is complete

### CODE REVIEW

**Comment by alvarlagerlof** [comment]
Created #18 for discussion

File: pages/adhd.mdx [hunk 1]
```diff
@@ -1,3 +1,64 @@
...
+/>
 
-TODO
+<Header>
+
+# What is ADHD?
+
+You've most likely heard of it. However, **most people don't actually know what it is.** No worries! We’ll clear that up here.
+
+OCD stands for **Attention deficit hyperactivity disorder**. But what does that mean? ADHD is [defined](https://www.ncbi.nlm.nih.gov/books/NBK519704/table/ch3.t13) by the [DSM-5](https://en.wikipedia.org/wiki/DSM-5) as:
```
Comment (Toasterbirb) [inline pages/adhd.mdx:13]: "OCD" should probably be changed to "ADHD" here

**Review by Toasterbirb — APPROVED** [review]

File: pages/adhd.mdx (hunk 1 above)
Comment (alvarlagerlof) [inline pages/adhd.mdx:13]:
```suggestion
ADHD stands for **Attention deficit hyperactivity disorder**. But what does that mean? ADHD is [defined](https://www.ncbi.nlm.nih.gov/books/NBK519704/table/ch3.t13) by the [DSM-5](https://en.wikipedia.org/wiki/DSM-5) as:
```

**Comment by alvarlagerlof** [comment]
Wops

**Comment by alvarlagerlof** [comment]
Closing this because we should figure out #41 first

**Pull request closed without merging**